plugins/outputs/warp10/warp10.go

```diff
@@ -0,0 +1,166 @@
+package warp10
+
+import (
+	"bytes"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"sort"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/plugins/outputs"
+)
+
+type Warp10 struct {
+	Prefix string
+
+	WarpUrl string
+
+	Token string
+}
+
+var sampleConfig = `
+  # prefix for metrics class Name
+  prefix = "telegraf."
+  ## POST HTTP Mode ##
+  # Url name of the Warp 10 server
+  warp_url = "localhost:4242/"
+  # Token to access your app on warp 10
+  token = "Token"
+`
+
+type MetricLine struct {
+	Metric    string
+	Timestamp int64
+	Value     string
+	Tags      string
+}
+
+func (o *Warp10) Connect() error {
+	return nil
+}
+
+func (o *Warp10) Write(metrics []telegraf.Metric) error {
+	if len(metrics) == 0 {
+		return nil
+	}
+	var timeNow = time.Now()
+	collectString := make([]string, 0)
+	index := 0
+	for _, mm := range metrics {
+
+		for k, v := range mm.Fields() {
+
+			metric := &MetricLine{
+				Metric:    fmt.Sprintf("%s%s", o.Prefix, mm.Name()+"."+k),
+				Timestamp: timeNow.Unix() * 1000000,
+			}
+
+			metricValue, buildError := buildValue(v)
+			if buildError != nil {
+				fmt.Printf("Warp: %s\n", buildError.Error())
+				continue
+			}
+			metric.Value = metricValue
+
+			tagsSlice := buildTags(mm.Tags())
+			metric.Tags = fmt.Sprint(strings.Join(tagsSlice, ","))
+
+			messageLine := fmt.Sprintf("%v// %s{%s} %v \n", metric.Timestamp, metric.Metric, metric.Tags, metric.Value)
```

Comment:
metric.Timestamp is an int & metric.Value is a string, so this should be (also why the space before the newline??):

```go
messageLine := fmt.Sprintf("%d// %s{%s} %s\n", metric.Timestamp, metric.Metric, metric.Tags, metric.Value)
```